portfolio/src/main/webapp/index.html

```diff
@@ -114,6 +114,10 @@ <h2>Attention</h2>
       The name is subject to change to become more interesting.
     </p>
   </div>
+  <div class="section" id="comments">
```

Comment:
Do you still need id = comments?

Reply:
I do not for this iteration! I had forgotten to take all those out, thanks for the reminder :)